docs/reference/api-conventions.asciidoc

```diff
@@ -378,6 +380,21 @@ supported units are:
 `tb`::  Terabytes
 `pb`::  Petabytes
 
+[[size-units]]
+[float]
+=== Data size units
```

Comment:
I meant this as a conversational question more than a command. I think "data size units" is unclear. I've hear people call them "quantities" before.